projects/aca-content/folder-rules/src/manage-rules/manage-rules.smart-component.spec.ts

```diff
@@ -204,18 +229,15 @@ describe('ManageRulesSmartComponent', () => {
     expect(component).toBeTruthy();
 
     const matProgressBar = loader.getHarness(MatProgressBarHarness);
-    const rules = debugElement.query(By.css('.aca-rule-list-item'));
-    const ruleDetails = debugElement.query(By.css('aca-rule-details'));
+    const rules = unitTestingUtils.getByCSS('.aca-rule-list-item');
+    const ruleDetails = unitTestingUtils.getByCSS('aca-rule-details');
```

Comment:
aca-rule-details is selector of component so please use getByDirective function instead to get that element